build.gradle

```diff
@@ -5,8 +5,8 @@ apply plugin: 'java'
 group = 'com.github.graycat27'
 version = '0.0.6'
 
-sourceCompatibility = '1.8'
-targetCompatibility = '1.8'
+sourceCompatibility = "11"
+targetCompatibility = "11"
```

Comment:
Gsonで用いる「FileWriter」コンストラクタのバージョン問題への対応
https://docs.oracle.com/javase/jp/11/docs/api/java.base/java/io/FileWriter.html#%3Cinit%3E(java.io.File,java.nio.charset.Charset)
＠since 11